docs/specification/draft/basic/index.mdx

```diff
@@ -116,3 +116,63 @@ There is also a
 [JSON Schema](https://github.com/modelcontextprotocol/specification/blob/main/schema/draft/schema.json),
 which is automatically generated from the TypeScript source of truth, for use with
 various automated tooling.
+
+## JSON Schema Usage
+
+The Model Context Protocol uses JSON Schema for validation throughout the protocol. This section clarifies how JSON Schema should be used within MCP messages.
+
+### Schema Dialect
+
+MCP supports JSON Schema with the following rules:
+
+1. **Default dialect**: When a schema does not include a `$schema` field, it defaults to JSON Schema draft-07 (`http://json-schema.org/draft-07/schema#`)
+2. **Explicit dialect**: Schemas MAY include a `$schema` field to specify a different dialect
+3. **Supported dialects**: Implementations SHOULD support at least draft-07 and SHOULD document which additional dialects they support
```

Comment:
```suggestion
3. **Supported dialects**: Implementations MUST support at least draft-07 and SHOULD document which additional dialects they support
```

I think this should be a MUST, as Draft 7 is the officially supported one by MCP, and otherwise implementations wouldn't work without it?